CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Confluent's Golang client for Apache Kafka
 
+## v1.9.3
+
+This is a maintenaince release:
+
+ * Added SetSaslCredentials. This new method (on the Producer, Consumer, and
+   AdminClient) allows modifying the stored SASL PLAIN/SCRAM credentials that
+	 will be used for subsequent (new) connections to a broker.
```

Comment:
nit: weird indent, align with previous line.

Reply:
Done and done